app/db.py

```diff
@@ -1,11 +1,12 @@
 import psycopg2
 import os
 class Connection(object):
+    """ creates connection to the database """
 
     def __init__(self):
         # ########### OFFICIAL DB CREDENTIALS ##############
-        self.HOSTNAME = 'localhost'
-        self.USERNAME = 'postgres'
+        self.HOSTNAME = 'ec2-50-16-241-91.compute-1.amazonaws.com'
```

Comment:
Get this from the environment variable too. It is especially dangerous to commit database credentials to Git.